.github/workflows/automated-release.yml

```diff
@@ -62,7 +63,14 @@ jobs:
                   DIST_REGISTRY: https://registry.npmjs.org/
 
             - name: Release with Gradle automatic
-              run: ./gradlew release -x test -x checkstyleTest -Prelease.useAutomaticVersion=true -Prelease.scope=${{ github.event.inputs.scope || env.DEFAULT_SCOPE }} -Pzowe.deploy.username=$ARTIFACTORY_USERNAME -Pzowe.deploy.password=$ARTIFACTORY_PASSWORD -Partifactory_user=$ARTIFACTORY_USERNAME -Partifactory_password=$ARTIFACTORY_USERNAME
+              shell: bash
+              run: |
+                  ./gradlew release -x test -x checkstyleTest -Prelease.useAutomaticVersion=true -Prelease.scope=${{ github.event.inputs.scope || env.DEFAULT_SCOPE }} -Pzowe.deploy.username=$ARTIFACTORY_USERNAME -Pzowe.deploy.password=$ARTIFACTORY_PASSWORD -Partifactory_user=$ARTIFACTORY_USERNAME -Partifactory_password=$ARTIFACTORY_USERNAME
```

Comment:
`Partifactory_password=$ARTIFACTORY_USERNAME`?

Reply:
well this line is unchanged, but good find, I'll update it I guess, it's working anyway so I suppose it picks the right password somewhere else.